claim/netdata-claim.sh.in

```diff
@@ -101,36 +101,48 @@ ID="unknown"
 ROOMS=""
 HOSTNAME=$(hostname)
 CLOUD_CERTIFICATE_FILE="${CLAIMING_DIR}/cloud_fullchain.pem"
+VERBOSE=0
+INSECURE=0
 
 # get the MACHINE_GUID by default
 if [ -r "${MACHINE_GUID_FILE}" ]; then
-	ID="$(cat "${MACHINE_GUID_FILE}")"
+        ID="$(cat "${MACHINE_GUID_FILE}")"
 fi
 
 # get token from file
 if [ -r "${CLAIMING_DIR}/token" ]; then
-	TOKEN="$(cat "${CLAIMING_DIR}/token")"
+        TOKEN="$(cat "${CLAIMING_DIR}/token")"
 fi
 
 # get rooms from file
 if [ -r "${CLAIMING_DIR}/rooms" ]; then
-	ROOMS="$(cat "${CLAIMING_DIR}/rooms")"
+        ROOMS="$(cat "${CLAIMING_DIR}/rooms")"
 fi
 
 for arg in "$@"
 do
-	case $arg in
-		-token=*) TOKEN=${arg:7} ;;
-		-url=*) URL_BASE=${arg:5} ;;
-		-id=*) ID=${arg:4} ;;
-		-rooms=*) ROOMS=${arg:7} ;;
-		-hostname=*) HOSTNAME=${arg:10} ;;
-		*)  echo >&2 "Unknown argument ${arg}"
-		    exit 1 ;;
-	esac
-	shift 1
+        case $arg in
+                -token=*) TOKEN=${arg:7} ;;
+                -url=*) URL_BASE=${arg:5} ;;
+                -id=*) ID=${arg:4} ;;
+                -rooms=*) ROOMS=${arg:7} ;;
+                -hostname=*) HOSTNAME=${arg:10} ;;
+                -verbose) VERBOSE=1 ;;
+                -insecure) INSECURE=1 ;;
+                -proxy=socks*) PROXY=${arg:7} ;;
```

Comment:
Does this mean we only support socks?

Reply:
yes, for now next PR will change this and bring support for others